app/views/events/_form.html.erb

```diff
@@ -29,4 +30,19 @@
   <div class="actions">
     <%= form.submit "Upload" %>
   </div>
+
+  <script>
```

Comment:
Script is required here. If put in application.html.erb, the script cannot find the <div> with id="editor". But I only tested this once so it should be tested again